client/injection/apiextensions/reconciler/apiextensions/v1beta1/customresourcedefinition/reconciler.go

```diff
@@ -317,6 +318,10 @@ func (r *reconcilerImpl) updateStatus(existing *v1beta1.CustomResourceDefinition
 			return nil
 		}
 
+		if diff, err := kmp.SafeDiff(existing.Status, desired.Status); err == nil && diff != "" {
+			logging.FromContext(ctx).Debugf("Updating status with: %s", diff)
```

Comment:
We may want to include the GVK or some identifier for the resource in the message?

Reply:
The logger is already decorated with the key, and this should print sufficient enclosing context about the Go type to determine the type information.  Maybe we should decorate the logger with `gvk: <foo>` independent of this if that's useful...?

Reply:
FYI the key decoration is generally how logstream works, which is a big reason I want this :)

Reply:
/lgtm